DoomModLoader2C/Forms/FormMod.cs

```diff
@@ -152,7 +162,10 @@ private void cmdPlay_Click(object sender, EventArgs e)
             pro.StartInfo.FileName = sourcePort.path;
             pro.Start();
 
-
+            if (WasStartedSuccessfullyFromCli())
```

Comment:
This way the workaround for the custom gzdoom quicksave (#14) is broken when starting from commandline, as the fix gets applied after the sourceport's process closes, that's why I was referring to the `OnSourceportClosed` method in this comment https://github.com/Premo36/DML2.X/pull/33#discussion_r1566388207. Moving the whole logic under that method would probably be a bit "wasteful" in terms of resources as it would keep dml always open until the sourceport closes, which is not really needed if the workaround is disabled.

One possible solution I can think of is to update this check so that we exit right away only when the workaround is not enabled
```suggestion
            if (!SharedVar.GZDOOM_QUICKSAVE_FIX && WasStartedSuccessfullyFromCli())
```

And at the end of the `OnSourceportClosed`, after the workaround code add
```
if (WasStartedSuccessfullyFromCli())
{
    Environment.Exit(0);
}
```

This way if there is no need to apply the workaround, we quit right away, otherwise we wait till the sourceport's close, the workaround code is executed and then we close

Reply:
Ah I'm with you, I've made your suggested changes and everything still works as expected and should mean the quicksave fix gets applied when enabled